pages/user guides/tidal-vms.md

```diff
@@ -1,30 +1,30 @@
 ---
 title: Pre-Configured Virtual Machines
 keywords: vm, virtual, machine, image, ova, ovf, aws, ami, tidal, tools
-last_updated: November, 2021
+last_updated: January, 2022
 summary: "Virtual machines pre-configured with Tidal software"
 sidebar: main_sidebar
 permalink: pre-configured-vms.html
 ---
 
-To help you get your Tidal Migrations discovery phase up and running, we've created a series of pre-configured virtual machines with common operating systems. These VMs have the latest Tidal software and dependencies installed for you, so you can focus on getting things done.
+To help you get your Tidal Migrations discovery phase up and running, we've created a pre-configured virtual machine images. These VMs have the latest Tidal softwares and dependencies installed for you, so you can focus on getting things done.
```

Comment:
Small grammar note, I think here you want `machine image` and `software` singular. Also `This VM has` etc if we're talking about a single one.

Reply:
Oh I just saw @przmv 's comment! Great minds think alike lol